public/app/features/provisioning/JobStatus.tsx

```diff
@@ -112,7 +112,10 @@ function RepositoryLink({ name }: RepositoryLinkProps) {
     return null;
   }
 
-  const repoHref = repo.spec?.github?.url;
+  const repoHref =
```

Comment:
Move this into a function: https://github.com/grafana/grafana/pull/96329/commits/8edf3af2a03d36a71afae639b9d565ab9b67f8e9